graphstore/triplestore_info.md

```diff
@@ -0,0 +1,138 @@
+# Gene Ontology Public Triple Store
+
+The Gene Ontology hosts an RDF triple store that contains
+[GO-CAM][1] data. This will outline what all is contained in more detailed in
+this store, the data models used, and how to query the store.
+
+The database is a [Blazegraph](https://www.blazegraph.com/) triple store.
+
+## Triple Store Contents
+
+The triple store contains data from GAFs from the model organism databases, and
+contains Production Noctua models. Additionally, it has the full Ontology.
+
+### The Pipeline
+
+The Blazegraph triple store gets produced as one of the steps of the pipeline.
+After the GAF annotations have been validated by [Ontobio](https://github.com/biolink/ontobio)
+the validated GAFs are loosely converted into GO-CAMs. There is roughly
+one "Annoton" per GAF annotation line. Once we have RDF GO-CAM of the GAFs,
+they are imported into the triple store. The specifics of how the this is
+organized in the triple store is discussed below.
+
+### Contents
+
+The Ontolgoy and all imports, all GAFs as GO-CAMs, and all production Noctua
```

Comment:
sp

Reply:
No see, Ontolgoy is the joy of working on Ontologies.